packages/cardpay-sdk/sdk/utils/conversions.ts

```diff
@@ -136,7 +136,7 @@ export async function getUsdConverter(
     return (amountInWei: BigNumber) => amountInWei;
   }
 
-  let rate = await tokenPairRate(provider, tokenAddress, usdcTokenAddress);
+  let rate = await tokenPairRate(provider, usdcTokenAddress, tokenAddress);
```

Comment:
Was this the reason for this bug? Was the rate inverted?

Reply:
This is one of the reasons, the other one is that I was not converting to the biggest unit.

Reply:
Ok. It's so weird we didn't notice this when we tested - I suppose because the rates are so odd on Goerli 🙈

Reply:
I was definitely running into this yesterday, and super confused about whether it was a) a bug in the converter code, b) Goerli wackiness, c) test setup mistakes, or d) a bug in my code. Thanks for fixing this.